src/js/common/model/veteran.js

```diff
@@ -775,13 +775,18 @@ function veteranToApplication(veteran) {
       case 'cityOfBirth':
       case 'stateOfBirth':
       case 'email':
+        if (value.value === '') {
+          return undefined;
+        }
+        break;
+
       case 'homePhone':
       case 'mobilePhone':
       case 'spousePhone':
         if (value.value === '') {
           return undefined;
         }
-        break;
+        return value.value.replace(/[- )(]/g, '');
```

Comment:
Yep, makes sense, just think we want to use the same validation function everywhere, so might need to make this change elsewhere too.